### ISSUE TITLE
Analyse CSKG Notebook: broken pipe errors while running zcat commands, kgtk queries and head commands

### ISSUE THREAD

**Issue by jaleedkhan** [usc-isi-i2/cskg#15]
Issues while running [Analyse CSKG Notebook](https://github.com/usc-isi-i2/cskg/blob/master/ESWC2021/Analyze%20CSKG.ipynb):

- In the first cell in that sets up environment variables and paths, the files mentioned in the code do not exist. I changed those paths to other files available in the [shared drive folder](https://drive.google.com/drive/u/1/folders/16347KHSloJJZIbgC9V5gH7_pRx0CzjPQ)
kg = "cskg_connected.kgtk.gz" (changed to “cskg.tsv.gz”)
nkg = "cskg-normalized.kgtk.gz" (changed to “cskg_connected_normalized.tsv.gz”)
- The zcat commands, kgtk query and head commands do not give the desired results due to several errors including broken pipe, as shown in [screencapture3.pdf](https://github.com/usc-isi-i2/cskg/files/7111757/screencapture3.pdf) attached herewith.

**Comment by filievski** [comment]
You are right, the filenames were misleading, and some of the commands did not run well.

The errors should be fixed now in the [same notebook](https://github.com/usc-isi-i2/cskg/blob/master/ESWC2021/Analyze%20CSKG.ipynb):
- Note: CSKG in KGTK format should be downloaded from here first https://drive.google.com/file/d/1mFwDJxav0H4CtA9UoPiwRHvPk_Zi4h0d/view?usp=sharing, it is the only input file that this notebook needs
- the zcat commands are now replaced with kgtk kypher commands
- the other commands are revised and tested on OSX

Let me know whether this resolves this issue on your end as well.

Thanks,
Filip

**Comment by jaleedkhan** [comment]
Hi Filip, the kgtk/kypher commands are not working anymore (shown in the screenshot below) after this update, including the ones in this "ESWC2021/Analyse CSKG" notebook. Everything is in place including the CSKG in KGTK format. I wonder if any changes have been made to kgtk (syntax, query format or anything)? 
![image](https://user-images.githubusercontent.com/71158275/134732867-6a970edf-a954-41e5-903b-0a19513a76b8.png)

**Comment by filievski** [comment]
Hi Jaleed, 
Here is what I'd suggest:

1. see whether `kgtk -h​` works - this should print a list of kgtk commands. If it does work, then I'd suggest replacing the variable `$kypher`​ with `kgtk query`​ in the notebook.
2. If kgtk​ does not work - I'd suggest starting with a fresh conda environment, installing `kgtk` and only the packages from CSKG that you need to run the specific notebooks. This is less convenient than using the `requirements.txt` file, but it is probably better, as the requirements file is quite complex, and as you observed, it could easily lead to a clash in versions.
